server/rpc_router.go

```diff
@@ -505,6 +507,17 @@ func (router *router) Subscribe(s Subscriber) error {
 }
 
 func (router *router) ProcessMessage(ctx context.Context, msg Message) error {
+	var err error
```

Comment:
should define in named return value. it will doesn't work when defer recover.
```
func (router *router) ProcessMessage(ctx context.Context, msg Message) (err error) {
```